src/Sulu/Bundle/SecurityBundle/Resources/js/views/Form/tests/toolbarActions/EnableUserToolbarAction.test.js

```diff
@@ -148,13 +148,23 @@ test('Return item config with loading button during request', () => {
     toolbarAction.resourceFormStore.resourceStore.data.id = 1234;
     toolbarAction.resourceFormStore.resourceStore.data.enabled = false;
 
-    expect(toolbarAction.getToolbarItemConfig().loading).toBeFalsy();
+    expect(toolbarAction.getToolbarItemConfig()).toEqual(expect.objectContaining({
+        loading: false,
+    }));
 
-    toolbarAction.getToolbarItemConfig().onClick();
-    expect(toolbarAction.getToolbarItemConfig().loading).toBeTruthy();
+    const toolbarItemConfig = toolbarAction.getToolbarItemConfig();
+    expect(toolbarItemConfig).not.toBeFalsy();
+    // $FlowFixMe
+    toolbarItemConfig.onClick();
+
+    expect(toolbarAction.getToolbarItemConfig()).toEqual(expect.objectContaining({
+        loading: true,
+    }));
 
     return deleteDraftPromise.then(() => {
```

Comment:
Not in this PR, but this promise still has a strange name :see_no_evil:

Reply:
already changed this in #4642 🙂